src/Sulu/Bundle/ContactBundle/Contact/ContactManager.php

```diff
@@ -394,7 +395,7 @@ public function getById($id, $locale)
     {
         $contact = $this->contactRepository->find($id);
         if (!$contact) {
-            return;
+            throw new EntityNotFoundException($this->contactRepository->getClassName(), $id);
```

Comment:
There are different ways to handle this, I would say the easiest way is to do it like in the [RoleController](https://github.com/sulu-io/sulu/blob/aa88f37a5a5560daa14ed71da2db794b82f646bb/src/Sulu/Bundle/SecurityBundle/Controller/RoleController.php#L276)